Cows-HacknSlash/Assets/NavMeshComponents/Editor/NavMeshComponentsGUIUtility.cs

```diff
@@ -0,0 +1,258 @@
+using UnityEngine;
+using UnityEngine.AI;
+
+namespace UnityEditor.AI
+{
+    public static class NavMeshComponentsGUIUtility
+    {
+        public static void AreaPopup(string labelName, SerializedProperty areaProperty)
+        {
+            var areaIndex = -1;
+            var areaNames = GameObjectUtility.GetNavMeshAreaNames();
+            for (var i = 0; i < areaNames.Length; i++)
+            {
+                var areaValue = GameObjectUtility.GetNavMeshAreaFromName(areaNames[i]);
+                if (areaValue == areaProperty.intValue)
```

Comment:
Please add curly braces to this if statement. We don't want to copy Apple's goto fail error!

Reply:
well i wouldn't dare to mess around with this code as these are built in library from unity